drivers/scsiutil.py

```diff
@@ -182,9 +182,8 @@ def scsi_dev_ctrl(ids, cmd):
             opterr='An error occured during the scsi operation')
 
 def getdev(path):
-    realpath = os.path.realpath(path)
```

Comment:
It turns out that calling realpath here meant that the match_dm check in the next line could never be true as any device of the form `/dev/mapper/xxxxx/` will realpath to something of the form `/dev/dm-xx`